migration/migration_blackbox_test.go

```diff
@@ -1,180 +1,100 @@
 package migration_test
 
 import (
-	"bufio"
-	"bytes"
 	"database/sql"
 	"fmt"
-	"html/template"
-	logger "log"
+	"os"
 	"testing"
 
-	config "github.com/fabric8-services/fabric8-cluster/configuration"
 	"github.com/fabric8-services/fabric8-cluster/migration"
-	"github.com/fabric8-services/fabric8-cluster/resource"
-	"github.com/fabric8-services/fabric8-common/log"
+	"github.com/fabric8-services/fabric8-common/gormsupport"
+	migrationsupport "github.com/fabric8-services/fabric8-common/migration"
+	"github.com/fabric8-services/fabric8-common/resource"
 
 	"github.com/jinzhu/gorm"
-	_ "github.com/lib/pq"
```

Comment:
don't you still need this import?

Reply:
Done in https://github.com/fabric8-services/fabric8-cluster/pull/24/commits/9127aea20f12b3acd503ae8b47efbec3c1852d8d